src/types/deputy.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2022 NowSecure Inc.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+export interface Deputy {
+  /**
+   * The version of Deputy.
+   */
+  deputy: string;
+  /**
+   * Name of the executable that was scanned.
+   */
+  package: string;
+  packageType: "iOS" | "STANDARD-APK" | "BUNDLE-APK";
+  /**
+   * SHA256 of the application package.
+   */
+  sha256: string;
```

Comment:
is this a lowercased sha256 such that we could make it explicit in the doc string, e.g., or is that not guaranteed.

Reply:
We make no guarantee but in practice it is lower-cased. However, this value is not used in this GitHub Action.